.github/dependabot.yml

```diff
@@ -0,0 +1,31 @@
+version: 2
+updates:
+  # Enable version updates for npm
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    open-pull-requests-limit: 10
+    reviewers:
+      - "o1-labs/o1js-maintainers"
+    labels:
+      - "dependencies"
+      - "javascript"
+    commit-message:
+      prefix: "npm"
```

Comment:
Not sure we need this commit prefix, but 🤷

Reply:
It should give something like: `npm(dev): bump @types/node from 18.x to 20.x`.